include/util/hex_float.h

```diff
@@ -709,6 +709,21 @@ inline std::istream& ParseNormalFloat(std::istream& is, bool negate_value,
   return is;
 }
 
+template <>
```

Comment:
Needs a blurb.  Actually, the previously existing function at line 700 is missing it, which could probably serve to document this one, since this one is just the specialization for the float16-based proxy.